lib/providers/circuit-breaker/dynamo.ts

```diff
@@ -0,0 +1,79 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient } from '@aws-sdk/lib-dynamodb';
+import Logger from 'bunyan';
+
+import { CircuitBreakerConfigurationProvider } from '.';
+import { BaseTimestampRepository, FillerTimestampMap, TimestampRepository } from '../../repositories';
+import { WebhookConfiguration } from '../webhook';
+
+export class DynamoCircuitBreakerConfigurationProvider implements CircuitBreakerConfigurationProvider {
+  private log: Logger;
+  private fillerHashes: string[];
+  private lastUpdatedTimestamp: number;
+  private timestampDB: BaseTimestampRepository;
+  private timestamps: FillerTimestampMap = new Map();
+  allow_list: Set<string> = new Set<string>();
```

Comment:
not using allow list here at all? also allow_list not super clear to me what its purpose is - comment would be helpful